robot-db/src/main/java/domain/Book.java

```diff
@@ -7,7 +7,11 @@
 
 
 /**
- * Book representation in database.
+ * {@link Book} is representation of book in a database.
```

Comment:
This is a very good JavaDoc comparing to that `interface for CRUD Operations on ...`. :+1: